impl-base/src/test/java/org/jboss/shrinkwrap/impl/base/exporter/StreamExporterTestBase.java

```diff
@@ -298,16 +304,16 @@ public void testExportThrowsArchiveExceptionOnAssetWriteFailure() throws IOExcep
             throw new RuntimeException("Mock Exception from an Asset write");
         }, PATH_ONE);
 
-        // Export
-        final InputStream in = this.exportAsInputStream(archive);
-
-        // Read in the full content (to in turn empty the underlying buffer and ensure we complete)
-        final OutputStream sink = new OutputStream() {
-            @Override
-            public void write(int b) {
-            }
-        };
-        Assertions.assertThrows(ArchiveExportException.class, () -> IOUtil.copyWithClose(in, sink));
+        try (// Export
```

Comment:
I believe this would look better as:

`try ( final InputStream in = this.exportAsInputStream(archive); // Export`